slasher/detection/attestations/spanner.go

```diff
@@ -64,11 +68,21 @@ func (s *SpanDetector) DetectSlashingsForAttestation(
 	defer traceSpan.End()
 	sourceEpoch := att.Data.Source.Epoch
 	targetEpoch := att.Data.Target.Epoch
-	if (targetEpoch - sourceEpoch) > params.BeaconConfig().WeakSubjectivityPeriod {
+	dis := targetEpoch - sourceEpoch
+
+	if sourceEpoch > targetEpoch { //Prevent underflow and handle source > target slashable cases.
```

Comment:
why flip source and target epoch here ?

Reply:
spans are recorded at source epoch. we need to get the right spans for detection to work right